.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: "Bug Report"
+about: "Report a bug to help us improve"
+title: "[Bug] "
+labels: ["bug"]
+assignees: []
+---
+
+### Description  
+<!-- A clear and concise description of the bug. -->
```

Comment:
Sorry if I wasn't clear, I meant uncomment _all_ of these. They're good notes, but when they're commented, people won't see them.